design_system/src/main/java/org/the_chance/honymart/ui/composables/HoneyAuthHeader.kt

```diff
@@ -1,30 +1,45 @@
 package org.the_chance.honymart.ui.composables
 
-import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.material3.MaterialTheme
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.text.style.TextAlign
-import androidx.compose.ui.unit.dp
 import org.the_chance.honymart.ui.theme.Typography
+import org.the_chance.honymart.ui.theme.black60
+import org.the_chance.honymart.ui.theme.black87
+import org.the_chance.honymart.ui.theme.dimens
 
 /**
  * Created by Aziza Helmy on 8/9/2023.
  */
 @Composable
-fun HoneyAuthHeader(title: String, subTitle: String, textColor: Color, modifier: Modifier = Modifier) {
-    Text(
-        text = title,
-        style = Typography.headlineMedium.copy(color = textColor),
-        textAlign = TextAlign.Center,
-        modifier = modifier
+fun HoneyAuthHeader(
+    title: String,
+    subTitle: String,
+    modifier: Modifier = Modifier,
+    titleColor: Color = black87,
```

Comment:
How change color with darkmode

Reply:
the dark mode is not provided yet so it is designed according to the provided light design.